workers/unity/Packages/com.improbable.gdk.core/Logging/LoggingDispatcher.cs

```diff
@@ -5,13 +5,21 @@
 namespace Improbable.Gdk.Core
 {
     /// <summary>
-    ///     Logs logEvents to console.
+    ///     Logs to Unity console.
```

Comment:
to Unity console -> to the Unity Console

I believe Laura decided that all Unity terms have to be capitalized